ecosystem/sf-indexer/aptos-sf-indexer/src/models/ledger_infos.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+#![allow(clippy::extra_unused_lifetimes)]
+use crate::schema::ledger_infos;
+
+#[derive(Debug, Identifiable, Insertable, Queryable)]
+#[diesel(table_name = "ledger_infos")]
+#[primary_key(chain_id)]
+pub struct LedgerInfo {
+    pub chain_id: i64,
```

Comment:
this is a `u8` afaik

Reply:
yea but it says that it can become a u64? 
https://github.com/aptos-labs/aptos-core/blob/main/types/src/chain_id.rs#L64-L65